docs/TROUBLESHOOTING.md

```
@@ -20,13 +20,61 @@ Common errors and solutions for Cortex Linux.
 ### Error: "No API key found"
 
 **Symptom:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)